### PR TITLE
German translations

### PR DESCRIPTION
**Pull request by StealWonders**
Follow up of https://github.com/VelocityPowered/Velocity/pull/261

This pull request basically implements all the provided strings in German. 
If other fluent speakers have any ideas for better translations I'm open for discussion :-)

### CODE REVIEW

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+# Commands
+velocity.command.generic-error=Beim Ausführen des Befehls ist ein Fehler aufgetreten.
+velocity.command.command-does-not-exist=Dieser Befehl existiert nicht.
+
+velocity.command.players-only=Nur Spieler können diesen Befehl ausführen.
+velocity.command.server-does-not-exist=Der angegebene Server {0} existiert nicht.
+
+velocity.command.server-current-server=Du bist derzeit verbunden mit {0}.
+velocity.command.server-too-many=Es sind zu viele Server eingerichtet. Verwenden Sie die Tabulatorvervollständigung, um alle verfügbaren Server aufzulisten.
```
Comment (kennytv) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties]: This is the only place you used `Sie` and I've never seen anyone actually use the full word "Tabulator" 👀.
```suggestion
velocity.command.server-too-many=Es sind zu viele Server eingerichtet. Verwende die Tabvervollständigung, um alle verfügbaren Server aufzulisten.
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+velocity.error.already-connected=Du bist bereits mit diesem Server verbunden!
+velocity.error.already-connected-proxy=Du bist bereits mit diesem Proxy verbunden!
+velocity.error.already-connecting=Du versuchst bereits eine Verbindung mit dem Server herzustellen!
+velocity.error.cant-connect=Keine Verbindung möglich zu {0}: {1}
+velocity.error.connecting-server-error=Es ist derzeit nicht möglich eine Verbindung mit {0} herzustellen. Bitte versuche es später erneut.
+velocity.error.connected-server-error=Bei der Verbindung zu {0} ist ein Problem aufgetreten.
+velocity.error.internal-server-connection-error=Bei der Verbindung mit dem Server ist ein interner Fehler aufgetreten.
+velocity.error.logging-in-too-fast=Du meldest dich zu schnell an, versuche es später noch einmal.
+velocity.error.online-mode-only=Du bist nicht in deinem Minecraft Konto eingeloggt. Wenn du in deinem Minecraft Konto eingeloggt bist, versuche dein Minecraft Client neu zu starten.
```
Comment (kennytv) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties]:
```suggestion
velocity.error.online-mode-only=Du bist nicht in deinem Minecraft Konto eingeloggt. Wenn du in deinem Minecraft Konto eingeloggt bist, versuche deinen Minecraft Client neu zu starten.
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+velocity.command.players-only=Nur Spieler können diesen Befehl ausführen.
+velocity.command.server-does-not-exist=Der angegebene Server {0} existiert nicht.
+
+velocity.command.server-current-server=Du bist derzeit verbunden mit {0}.
+velocity.command.server-too-many=Es sind zu viele Server eingerichtet. Verwenden Sie die Tabulatorvervollständigung, um alle verfügbaren Server aufzulisten.
+velocity.command.server-available=Verfügbare Server:
+velocity.command.server-tooltip-player-online={0} spieler online
+velocity.command.server-tooltip-players-online={0} spieler online
+velocity.command.server-tooltip-current-server=Du bist derzeit mit diesem Server verbunden
+velocity.command.server-tooltip-offer-connect-server=Klicke, um sich mit diesem Server zu verbinden
```
Comment (kennytv) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties]:
```suggestion
velocity.command.server-tooltip-offer-connect-server=Klicke, um dich mit diesem Server zu verbinden
```

**Review by kennytv — CHANGES_REQUESTED** [review]

**Review by kennytv — APPROVED** [review]
👍

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+velocity.command.generic-error=Beim Ausführen des Befehls ist ein Fehler aufgetreten.
+velocity.command.command-does-not-exist=Dieser Befehl existiert nicht.
+
+velocity.command.players-only=Nur Spieler können diesen Befehl ausführen.
+velocity.command.server-does-not-exist=Der angegebene Server {0} existiert nicht.
+
+velocity.command.server-current-server=Du bist derzeit verbunden mit {0}.
+velocity.command.server-too-many=Es sind zu viele Server eingerichtet. Verwenden Sie die Tabulatorvervollständigung, um alle verfügbaren Server aufzulisten.
+velocity.command.server-available=Verfügbare Server:
+velocity.command.server-tooltip-player-online={0} spieler online
```
Comment (aurorasmiles) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties]:
```suggestion
velocity.command.server-tooltip-player-online={0} Spieler online
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+velocity.command.server-tooltip-players-online={0} spieler online
```
Comment (aurorasmiles) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties]:
```suggestion
velocity.command.server-tooltip-players-online={0} Spieler online
```

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+velocity.command.server-tooltip-current-server=Du bist derzeit mit diesem Server verbunden
+velocity.command.server-tooltip-offer-connect-server=Klicke, um sich mit diesem Server zu verbinden
+
+velocity.command.glist-player-singular=Spieler
+velocity.command.glist-player-plural=Spieler
+velocity.command.glist-total-suffix= sind aktuell mit dem Proxy verbunden.
+velocity.command.glist-view-all=Um alle Spieler auf Servern aufzulisten, verwende /glist all.
+
+velocity.command.reload-success=Velocity-Konfiguration erfolgreich neu geladen.
+velocity.command.reload-failure=Die Velocity-Konfiguration konnte nicht neu geladen werden. Prüfe die Konsole auf weitere Details.
```
Comment (aurorasmiles) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_de_DE.properties]:
```suggestion
velocity.command.reload-failure=Die Velocity-Konfiguration konnte nicht neu geladen werden. Prüfe die Konsole für weitere Details.
```

**Review by aurorasmiles — CHANGES_REQUESTED** [review]

**Comment by astei** [comment]
Can you update your translations with the new messages in 1e8454c7? Thanks!

**Review by aurorasmiles — APPROVED** [review]

**Comment by astei** [comment]
[Imported into Crowdin](https://crowdin.com/project/velocity/de)

**Pull request closed without merging**